docs/general/commands.md

```diff
@@ -188,3 +188,34 @@ $ npm run lint:css
 ```
 
 Only lints your CSS.
+
+## Docker
+First, you need to build your image. Make sure that you stand on the root directory of the project.
+Then, run the below command. You can specify the image name after -t (e.g. `myweb_image`)
+```
+docker build -t myweb_image .
+```
+If everything goes well, you can summon the docker image up by running below command.
+```
+docker run -d -p 3000:3000 --name myweb_container myweb_image
+```
+Now, you should be accessible to your very first time of this boilerplate by accessing `http://localhost:3000`
+
+However, you cannot made any change in that example app yet. We need to link docker container directory
+to what we have in our local machine. First, let's destroy the old container.
+```
+docker rm -f myweb_container
+```
+Then, let's run below command to create new container with a link directory to your local machine
+```
+docker run -it -p 3000:3000 -v {YOUR_PROJECT_ROOT_FULL_PATH}:/reactapp --name myweb_container myreact bash
```

Comment:
This might stump people - trying to run the `myreact` image but it was called `myweb_image` when it was built. Suggest changing one of them for consistency.